common/dna_data/wc_easter_egg_dna.txt

```diff
@@ -0,0 +1,1109 @@
+aliza_dna = { # by ercarp
```

Comment:
Encoding should be changed.
```
[22:45:58][lexer.cpp:332]: File 'common/dna_data/wc_easter_egg_dna.txt' should be in utf8-bom encoding (will try to use it anyways)
```

Reply:
Fixed in 7b2b72e109bcdea81d141edeb6ad82bdba68f8e4.